tests/pipeline/pipeline_tests.py

```diff
@@ -51,7 +51,7 @@ class PipelineTest(unittest.TestCase):
 
   @staticmethod
   def _create_context():
-    project_id = 'test'
+    project_id = 'test.project'
```

Comment:
Read project IDs can not contain a period, only lowercase letters, digits, or hyphens.

Reply:
Removed; thanks!